.gitignore

```diff
@@ -1,2 +1,5 @@
 __pycache__/
 .mypy_cache/
+venv/
+*.egg-info/
+build/
```

Comment:
This change is unrelated; I suggest to add those to your local `.git/info/exclude` to not have them listed as untracked files. (also see https://git-scm.com/docs/gitignore )